WrightTools/data/_variable.py

```diff
@@ -34,6 +34,7 @@ def __init__(self, parent, id, units=None, **kwargs):
             Additional keys and values to be written into dataset attrs.
         """
         self._parent = parent
+        super_kwargs = {k: kwargs[k] for k in ("compression", "shuffle") if k in kwargs}
```

Comment:
```suggestion
```

initially thought passing kwargs here was needed, turned out only in the _data file